tests/test_structures/test_datasample.py

```diff
@@ -5,12 +5,12 @@
 import torch
 from mmengine.structures import LabelData
 
-from mmcls.structures import ClsDataSample
+from mmcls.structures import ClsDataSample, MultiTaskDataSample
 
 
-class TestClsDataSample(TestCase):
 
-    def _test_set_label(self, key):
+class TestClsDataSample(TestCase):
+    def _test_set_label(self,key):
         data_sample = ClsDataSample()
```

Comment:
please checkout this part 

see https://stackoverflow.com/a/6963150

you should be able to do something like

```
git checkout -p my_multi_task_1_x-- tests/test_structures/test_datasample.py
```